### ISSUE TITLE
Changing distribution, Bernoulli, Laplace

### ISSUE THREAD

**Issue by CoteDave** [cpmpercussion/keras-mdn-layer#28]
Hi,

It would be vey usefull to be able to tweak the type of distribution in the customized loss function. For example, for a binary classification problem, changing the distribution to Bernoulli (Or any other strategy to be able to do classification). For regression, changing Normal distribution to Laplace, Exponential, LogNormal or Gamma distribution to see if the results are better.

I tried to make some modifications myself on your code, but i'm really not sure about the final result !

Thanks !
